exercises/largest-series-product/description.md

```diff
@@ -12,3 +12,5 @@ in the input; the digits need not be *numerically consecutive*.
 
 For the input `'73167176531330624919225119674426574742355349194934'`,
 the largest product for a series of 6 digits is 23520.
+
+For a series of zero digits, the largest product is 1 because 1 is the identity for multiplication.
```

Comment:
To discuss: Should a link be added? Candidates:

* https://en.wikipedia.org/wiki/Identity_element - note that this lists identities for operations other than multiplication as well.
* https://en.wikipedia.org/wiki/Empty_product - note that when reading the first paragraph you don't want to get mixed up between the additive identity 0 versus the multiplicative identity 1.

I'm not sure there should be a link here because I don't want to imply that reading the link is required to solve the problem. We could solve the problem by saying "You don't have to know what a multiplicative identity is, but if you wanted to, you could read X"